### PR TITLE
Simplify dev edition product, firstrun, and whatsnew pages (Fix #14243 Fix #14244 )

### LINKED ISSUE #14243: Simplify dev edition whatnew page
Make the whatsnew page very simple, link to the release notes.

### LINKED ISSUE #14244: Simplify dev edition firstrun page
Convert devedition product and firstrun pages base protocol and remove references to "new" features.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

We are simplifying these pages to be closer to base protocol in preparation for upcoming design system changes.

## Significant changes and points to review

- Remove references to features being "new"
- Remove custom styles sheets for all dev edition page includes
- Change icon colours to match dev edition pallet
- Convert highlights to card component
- Convert features to use picto component
- Convert newsletter to use split component
- Generic whats new page:
  - Add link to release notes
  - Remove feature list and replace with link to the docs
  - Darker theme
  - Add MDN component

## Issue / Bugzilla link

Fix #14243
Fix #14244 

## Testing

- Link to release notes identifies itself as in English only, when viewing in a language other than English
- Linux download information appears properly on product page
- http://localhost:8000/en-US/firefox/developer/
- http://localhost:8000/en-US/firefox/99.0a2/firstrun/
- http://localhost:8000/en-US/firefox/99.0a2/whatsnew/ *note: this does not replace the MDN Plus promo yet*

### CODE REVIEW

File: l10n/en/firefox/developer.ftl [hunk 1]
```diff
@@ -73,16 +73,24 @@ firefox-developer-firefox-developer-edition-sends = { -brand-name-firefox-develo
...
-
-# Obsolete string
-firefox-developer-all-the-latest = All the latest developer tools in beta, plus <strong>experimental features</strong> like the Multi-line Console Editor and WebSocket Inspector.
-
 firefox-developer-a-separate-profile = A <strong>separate profile and path</strong> so you can easily run it alongside Release or { -brand-name-beta } { -brand-name-firefox }.
 firefox-developer-preferences-tailored = Preferences <strong>tailored for web developers</strong>: Browser and remote debugging are enabled by default, as are the dark theme and developer toolbar button.
-firefox-developer-congrats-you-now-have = Congrats. You now have { -brand-name-firefox-browser } { -brand-name-developer-edition }.
-firefox-developer-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+firefox-developer-congrats-you-now-have = Congrats. You now have { -brand-name-firefox } { -brand-name-developer-edition }.
+firefox-developer-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox } { -brand-name-developer-edition }.
```
Comment (craigcook) [inline l10n/en/firefox/developer.ftl:82]: I think these probably need new strings IDs even if all that's changing are the brand name variables. We should check with Peiying to be sure, but I don't think locales will pick up the changes without a new ID (and we can use the old ones as fallbacks).

**Review by craigcook — CHANGES_REQUESTED** [review]

File: l10n/en/firefox/developer.ftl (hunk 1 above)
Comment (stephaniehobson) [reply l10n/en/firefox/developer.ftl]: Turns out only one is in use, so I versioned that one and deleted the other. I also added some l10n checks before displaying the new content.

**Review by craigcook — APPROVED** [review]
👩‍💻

File: l10n/en/firefox/developer.ftl [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 firefox-developer-page-title = { -brand-name-firefox-developer-edition }
 firefox-developer-firefox-developer-edition-desc = { -brand-name-firefox-developer-edition } is the blazing fast browser that offers cutting edge developer tools and latest features like CSS Grid support and framework debugging
-firefox-developer-firefox-browser = { -brand-name-firefox-browser } { -brand-name-developer-edition }
+firefox-developer-firefox-browser = { -brand-name-firefox } { -brand-name-developer-edition }
```
Comment (flodolo) [inline l10n/en/firefox/developer.ftl:11]: This change requires a new string ID

File: l10n/en/firefox/developer.ftl [hunk 3]
```diff
@@ -73,16 +73,26 @@ firefox-developer-firefox-developer-edition-sends = { -brand-name-firefox-develo
 firefox-developer-download-the-firefox-browser = Download the { -brand-name-firefox } browser made for developers
 firefox-developer-welcome-to-the-all-new = Welcome to the all-new { -brand-name-firefox-quantum }: { -brand-name-developer-edition }
 firefox-developer-firefox-has-been-rebuilt = { -brand-name-firefox } has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
-firefox-developer-congrats-you-now-have-firefox = Congrats. You now have { -brand-name-firefox-quantum }: { -brand-name-developer-edition }.
-firefox-developer-this-isnt-just-an-update = This isn’t just an update. This is { -brand-name-firefox-quantum }: A brand new { -brand-name-firefox } that has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
-firefox-developer-welcome-to-firefox-browser = Welcome to { -brand-name-firefox-browser } { -brand-name-developer-edition }
+firefox-developer-welcome-to-firefox-browser = Welcome to { -brand-name-firefox } { -brand-name-developer-edition }
```
Comment (flodolo) [inline l10n/en/firefox/developer.ftl:76]: This requires a new ID.

File: l10n/en/firefox/installer-help.ftl [hunk 4]
```diff
@@ -19,9 +19,9 @@ installer-help-firefox-release-title = { -brand-name-firefox-browser }
 #   $trackers (number) - number of trackers blocked by Firefox (currently in the thousands)
 installer-help-firefox-release-desc = Get the latest. Automatic privacy is here. Download { -brand-name-firefox-browser } to block over { $trackers } trackers.
 
-installer-help-firefox-beta-title = { -brand-name-firefox-browser } { -brand-name-beta }
+installer-help-firefox-beta-title = { -brand-name-firefox } { -brand-name-beta }
```
Comment (flodolo) [inline l10n/en/firefox/installer-help.ftl:22]: This need a new ID, as well as the next change.

File: l10n/en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl [hunk 5]
```diff
@@ -6,7 +6,7 @@
 
 # HTML page title
 developer-mdnplus-page-title = { -brand-name-firefox-developer-edition }
-developer-mdnplus-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+developer-mdnplus-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox } { -brand-name-developer-edition }.
```
Comment (flodolo) [inline l10n/en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl:9]: And this as well.

**Review by flodolo — COMMENTED** [review]
You cannot replace a term reference within a string without a new ID. That will trigger a bunch of errors, on top of no translation being updated (there is already a translation, using the wrong term reference).

**Comment by craigcook** [comment]
@flodolo Sorry, there was a miscommunication here... in my original review comment I meant to imply that ALL changed strings required a new ID, not only the one I commented on. When @stephaniehobson said "only one was in use," I took that to mean NONE of the other changed strings were in use at all and this PR would be adding them as new. So we misunderstood each other and I didn't clarify before I merged so that's on me. We'll get these fixed up. Sorry for the mistake.

**Pull request merged**